src/styles/main.scss

```diff
@@ -1,3 +1,58 @@
-body {
-  margin: 0;
+@import 'Card';
+@import 'Header';
+@import 'Stars';
+
+.card {
+  margin-right: 48px;
+  width: 200px;
+  height: 408px;
+  transition: transform 0.3s ease, box-shadow 0.3s ease;
+  margin-bottom:46px;
+}
```

Comment:
If you have created a separate file for `Card` styles, you must keep all the styles related to this block in that file.